### DISCUSSION TITLE
Add "port" to "options" like http[s].request

### DISCUSSION THREAD

**Discussion by tomate1** [node-fetch/node-fetch#1492] [Ideas]
Just a suggestion... I would like to know what others think about it.

**Comment by jimmywarting** [comment]
what is wrong with a url that includes a port?
You could alternative do: 

```js
const url = new URL('https://localhost')
Object.assign(url, { port: 3333 })

fetch(url)
```
A tiny bit verbose but gets the job done. 
port is not part of the spec and i want to keep such things away...

**Reply by tomate1** [reply]
Thank you for the kind reply.
And I didn't know about this possibility of assigning the port to the URL object. That works.
